.github/workflows/rust-windows.yml

```diff
@@ -0,0 +1,57 @@
+name: Rust CI (Windows)
```

Comment:
I prefer to have two jobs in one workflow. I think that will be easier to keep in sync.